lm75bd/lm75bd.c

```diff
@@ -26,8 +26,26 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
+  error_code_t errCode;
+  if (!temp) {
+    return ERR_CODE_INVALID_ARG;
+  }  
+
+  // Configure ptrReg to select tempReg.
+  uint8_t ptrReg = 0b00000000;
```

Comment:
Make sure to define a macro for the pointer address for readability. Refer to LM75BD_REG_CONF macro as an example on how it's done.

Reply:
Done.